_components/accordions.md

```diff
@@ -139,29 +139,21 @@ lead: Intro text on what is included in this section and how to use it. No more
     Documentation
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
-    <h4 class="usa-heading">Use</h4>
+    <h4 class="usa-heading">Usability</h3>
```

Comment:
Should this be an h4 or an h3? Right now you have `<h4>` as the opening tag and `</h3>` as the closing tag.